linebot/flex_test.go

```diff
@@ -127,6 +127,44 @@ func TestUnmarshalFlexMessageJSON(t *testing.T) {
 		},
 		{
 			JSON: []byte(`{
+	"type": "bubble",
+	"size": "nano",
+    "body": {
```

Comment:
Are tabs and spaces mixed here? It looks like we have tab indentation for "type" and "size" while others use spaces. Can we use spaces to follow other parts? This may look the same on the editor, but can be confusing when reviewing on p-r.

Reply:
Thank you, I didn't realize that. By the way the test case I referred to [this](https://github.com/line/line-bot-sdk-go/blob/b8c029f8c185e2c8bf51180afc5438740b2ae652/linebot/flex_test.go#L30) looks like it has 2 more space compared to other test case, should I make them all including the one I add have the same indentation ?

Reply:
Thank you for the offer. Yes, could you apply the same rule to https://github.com/line/line-bot-sdk-go/pull/178/files#diff-f686096bea3c90a3a39611d167986ec2R30-R45 ?
This part is not exactly what this p-r is focusing on, but I'd like to unify the rule within a single test.